kits/llm-response-qa/apps/.env.example

```
@@ -0,0 +1,3 @@
+LAMATIC_API_KEY=
+OPENROUTER_API_KEY=
+NEXT_PUBLIC_APP_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)